api/SVGZoomAndPan.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGZoomAndPan": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGZoomAndPan",
+        "support": {
+          "webview_android": {
```

Comment:
@maboa please put `webview_android` at the end of the browser list to keep it alphabetical, see #1882. Unfortunately this'll need to be fixed on all your PRs 😬